src/index.js

```diff
@@ -60,6 +65,17 @@ const downloadImages = () => {
 };
 
 /**
+ * Make sure the filenames are safe by stripping out problematic characters
+ * @param {String} filename a potential filename
+ * @returns {String} filename with characters stripped out
+ */
+const cleanFileName = function (filename) {
```

Comment:
String transformations shouldn't be too bad in terms of performance, even if you are doing it over and over. I'm pretty sure that downloading and saving on disk is a lot more expensive than that.

And about the reason of having this, it makes sense.